linera-sdk/src/contract/runtime.rs

```diff
@@ -378,6 +380,22 @@ where
         ApplicationId::from(application_id).with_abi::<Abi>()
     }
 
+    /// Creates a new data blob and returns its hash.
+    pub fn create_data_blob(&mut self, bytes: Vec<u8>) -> DataBlobHash {
+        let blob_id = contract_wit::create_data_blob(&bytes);
+        DataBlobHash(blob_id.hash.into())
+    }
+
+    /// Publishes a module with contract and service bytecode and returns the module ID.
+    pub fn publish_module(
```

Comment:
(I wonder if it would be better to have a `publish_wasm_module` with two and a `publish_evm_module` with only one argument? Maybe also on the CLI and in the node service. Let's discuss with the team.)